package.json

```diff
@@ -39,6 +39,7 @@
     "ed25519.js": "^1.3.0",
     "google-protobuf": "^3.9.1",
     "js-sha3": "^0.8.0",
+    "keccak": "^3.0.2",
```

Comment:
if it's required only for examples and `examples` is in `.npmignore` - it's better to put it into `devDependencies`

Reply:
Yes, made the changes accordingly.